tasks/bosh-create-env.sh

```diff
@@ -14,8 +14,11 @@ pushd state/environments/softlayer/director/$BOSH_LITE_NAME
         /tmp/bosh.yml \
         -v director_vm_prefix=$BOSH_LITE_NAME
 
-    tail -n1 /etc/hosts > hosts
-    cut -d ' ' -f1 < hosts > ip
+    hosts_entry=$(grep $BOSH_LITE_NAME /etc/hosts || true)
+    if [ -n "$hosts_entry" ]; then
```

Comment:
I think we should fail if we can't find the host entry. This bosh-lite won't be usable.

Reply:
Discussed offline. We'll use this version.